src/cephadm/cephadm.py

```diff
@@ -1005,9 +1005,12 @@ def __init__(self,
         self.image = image
 
         self.sock_dir = config_json.get('sock-dir', '/var/run/ceph/')
-        ipv4_addrs, ipv6_addrs = get_ip_addresses(get_hostname())
-        # use the first ipv4 (if any) otherwise use the first ipv6
-        addrs = next(iter(ipv4_addrs or ipv6_addrs), None)
+        addrs = None
```

Comment:
This line can be deleted.
```suggestion
        addrs = None
```